.cirrus/Dockerfile.ubuntu16.04

```diff
@@ -31,3 +29,25 @@ RUN set -e -x ;\
     cmake -DCMAKE_INSTALL_PREFIX=/usr . ;\
     make -j $(nproc) ;\
     make install
+
+RUN set -e -x ;\
```

Comment:
Please order the RUN blocks for installing in the following order:

- iverilog
- icestorm
- yosys
- trellis

(from least to most likely to be updated, so that we don't have to rebuild unnecessary layers)